### ISSUE TITLE
Wayland version Compiled for OpenSUSE Tumbleweed, get error when running

### ISSUE THREAD

**Issue by fearlessgeekmedia** [espanso/espanso#1530]
**Describe the bug**
When I try to run espanso, I get this error.

espanso: error while loading shared libraries: libwx_gtk3u_html-3.2.so.0: cannot open shared object file: No such file or directory

**To Reproduce**
1. Compile
2. Attempt to register Espanso, or run Espanso

**Expected behavior**
I expect it to ru

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**Logs**
If possible, run `espanso log` in a terminal after the bug has occurred, then post the output here so that we can better diagnose the problem

**Setup information**
- OS: OpenSUSE Tumbleweed (MicroOS)
- Version: 2.1.8

Additionally, I have libwx_gtk3u_html-suse8_0_0, libwx_gtk3u_html-suse-nostl8_0_0, and wxWidgets-3_2-devel installed.

**Comment by Vistaus** [comment]
Can't reproduce this on openSUSE Tumbleweed.

**Comment by j-lakeman** [comment]
~Can't reproduce either as of now.~
The package `espanso` from the main repo seems only to work in X sessions / X windows though.

**Comment by smeech** [comment]
> The package `espanso` from the main repo seems only to work in X sessions / X windows though.

That's not what I have found - the *only* times I have compiled were for Wayland.

Or perhaps I'm misunderstanding your comment!

**Comment by j-lakeman** [comment]
Sorry my misunderstanding here. I didn't compile the package myself but simply used the one from the official repo: https://software.opensuse.org/package/espanso
That one seems to only work in X sessions / windows.

**Comment by smeech** [comment]
Ah, thanks. I don't know how openSUSE grab their packages, but we maintain different versions for [X11 & Wayland](https://espanso.org/docs/install/linux/#choosing-the-right-install-method). I can see we don't have one for openSUSE, but expect people to compile, which takes us back to @fearlessgeekmedia's problem.

**Comment by smeech** [comment]
Possibly related: https://github.com/espanso/espanso/issues/2096#issuecomment-2413422321 and https://github.com/espanso/espanso/issues/1793, although there have been improvements to the source since the original Issue.

**Comment by fearlessgeekmedia** [comment]
This is an old problem and I overlooked the e-mail notifications. I'm not even on openSUSE anymore. These days I'm using Void Linux (musl). But I have not gotten Espanso to work in that on Wayland, either. It's not available in the Void repos either. I've tried installing espanso-waylnad with the Nix package manager and that didn't work for me on Void.

**Comment by smeech** [comment]
Sadly, Wayland support remains flaky. Compilation seems the most reliable way of installing Espanso, but that fails frequently.

**Comment by fearlessgeekmedia** [comment]
Yeah I've kind of given up on using Espanso on Wayland at this point. It's
great if I was sticking to X11, but most things I do can be done in Wayland
now. And that's the direction things are going. I check back in once in a
while to see if I can get it working. But no success as of yet.

**Comment by leggewie** [comment]
FWIW, I get the same problem on Ubuntu Noble with the 2.2.1 deb package from this project

this is a regression from 0.7.3

**Comment by AucaCoyan** [comment]
Hey everyone!
This was reported in https://github.com/espanso/espanso/issues/1793 , with fix instructions.
I know it's a different distro, but this issue is getting old and we want to mope up 👌.
I don't have a redhat-based distro, so if anybody is having problems, please comment in [the discord](https://discord.gg/DFcCNDg7bB) so somebody can follow up the conversation.
Have a nice day!
